tests/test_cli.py

```diff
@@ -42,23 +42,23 @@ def test_missing_argument(cli_runner):
 
 
 @pytest.mark.parametrize(
-    "command_line",
+    "command_line, expected_message",
     [
         # Missing options
-        ["run", "."],
-        # Too many options
-        ["run", "--removed-in", "3.0", "--deprecated-in", "2.0", "."],
+        (["run", "."], "No codemods were selected"),
+        # Invalid version
+        (
+            ["run", "--removed-in", "3.0", "--deprecated-in", "6.4", "."],
+            "is not supported",
+        ),
```

Comment:
Not sure we need this test parametrisation, there is already `test_non_supported_version` covering unsupported versions.